html/search/search.css

```
@@ -0,0 +1,377 @@
+/*---------------- Search Box positioning */
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)